$OpenBSD: patch-configure,v 1.9 2014/03/27 19:04:23 jasper Exp $

XXX: Workaround bug in pkg-config(1) where nothing is returned and it eats cpu.

--- configure.orig	Thu Mar 27 17:16:21 2014
+++ configure	Thu Mar 27 17:16:55 2014
@@ -16640,7 +16640,7 @@ fi
 if test "x$have_uoa" = "xyes" -o "x$have_goa" = "xyes"; then
    { $as_echo "$as_me:${as_lineno-$LINENO}: checking Mission Control plugins dir" >&5
 $as_echo_n "checking Mission Control plugins dir... " >&6; }
-   MISSION_CONTROL_PLUGINS_DIR=`pkg-config --define-variable=libdir='${libdir}' --variable=plugindir mission-control-plugins`
+   MISSION_CONTROL_PLUGINS_DIR=`pkg-config --variable=plugindir mission-control-plugins`
 
    { $as_echo "$as_me:${as_lineno-$LINENO}: result: $MISSION_CONTROL_PLUGINS_DIR" >&5
 $as_echo "$MISSION_CONTROL_PLUGINS_DIR" >&6; }
